pymc/distributions/continuous.py

```diff
@@ -1042,6 +1042,9 @@ class Beta(UnitContinuous):
        f(x \mid \alpha, \beta) =
            \frac{x^{\alpha - 1} (1 - x)^{\beta - 1}}{B(\alpha, \beta)}
 
+    where :math:`B(\alpha, \beta) = \frac{\Gamma(\alpha)\Gamma(\beta)}{\Gamma(\alpha + \beta)}`
```

Comment:
Is it enough to say `where B is the Beta distribution`?

Reply:
Seems OK to me as well.

I'm also wondering if it would be nice to link somewhere to clarify what the Beta function is, or maybe that's too elementary to bother?

Reply:
We can add a link to the Wikipedia page for the Neta distribution? From there users should be able to find the formula and specifics

Reply:
And thank you for the useful guidance!
(Just now I realized that you suggested to link to the distribution, not the function in the first place :smile: .)

I've added another commit to change to option 1.
(I guess you could squash them together before applying to `main`?)